### ISSUE TITLE
Client side validations are not working for student module 

### ISSUE THREAD

**Issue by srichatala** [COMP313Team4/UniversalTokenSytem-AndoridApplication#48]
Issues in Task #16. 

1)space should be accepted in first name and last name
2) Phone number formation should accept xxx-xxx-xxxx this format
3) If the validations enabled, should not give option to "submit" the form, Make "Update" button inactive 
4) By default submit button should be in inactive mode. Once user enter valid details then only "Submit" button should be in active mode 

![image](https://cloud.githubusercontent.com/assets/9619400/10643443/dfa32b98-77f0-11e5-8d00-da1d502fca1b.png)

![image](https://cloud.githubusercontent.com/assets/9619400/10643466/036a024a-77f1-11e5-8122-0cfdd3986af5.png)

**Comment by shivanimgupta** [comment]
e986ab7

**Comment by srichatala** [comment]
Hi Shivani,

Could you plz check the image that shows the validation for Course name.
![image](https://cloud.githubusercontent.com/assets/9619400/10671261/aa050a3e-78b7-11e5-95b9-a58b8f373821.png)

**Comment by Alagu91** [comment]
Yes, It is only accepting one word.. More than that, It is giving the
error..

On Thu, Oct 22, 2015 at 12:23 PM, SriChatala notifications@github.com
wrote:

> Hi Shivani,
> 
> Could you plz check the image that shows the validation for Course name.
> [image: image]
> https://cloud.githubusercontent.com/assets/9619400/10671261/aa050a3e-78b7-11e5-95b9-a58b8f373821.png
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/COMP313Team4/UniversalTokenSytem-AndoridApplication/issues/48#issuecomment-150279290
> .

## 

Thanks and Regards,
Alagu Murugappan
001-416-418-5780

**Comment by shivanimgupta** [comment]
e2045e0
